www/js/diary.js

```diff
@@ -352,6 +353,44 @@ angular.module('emission.main.diary',['ui-leaflet', 'nvd3ChartDirectives',
     };
     */
 
+    /* For UI control */
+    $scope.groups = [];
+    for (var i=0; i<10; i++) {
```

Comment:
This is dead code, right? I am going to merge for now and defer cleanup until next week.